interfaces/automated-testing/scd/scd.yaml

```diff
@@ -0,0 +1,72 @@
+openapi: 3.0.2
+info:
+  title: Strategic Conflict Detection Test Data Injection
+  version: 0.0.1
+  description: >-
+    This interface is implemented by every USS wishing to be tested by the automated testing framework.
+    The automated testing suite calls this interface to inject test data into the USS system under test.
+
+components:
+  securitySchemes:
+    Authority:
+      type: oauth2
+      flows:
+        clientCredentials:
+          tokenUrl: https://auth.example.com/oauth/token
+          scopes:
+            utm.inject_test_data: |-
+              Client may inject test data into a USS for the purpose of conducting automated tests.
+      description: |-
```

Comment:
I think it's less than ideal to copy the full content of SCD auth description here; instead, can we basically say that these are SCD auth tokens with extended scopes?  [Here](https://github.com/interuss/dss/blob/744b7a4733c785066802533b9aa80485fc68db62/interfaces/automated-testing/rid/injection.yaml#L176) is how the RID testing interface does it.